com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -2065,6 +2065,20 @@ internal void HandleConnectionApproval(ulong ownerClientId, ConnectionApprovalRe
 
                     SendMessage(ref message, NetworkDelivery.ReliableFragmentedSequenced, ownerClientId);
 
+                    for (int index = 0; index < MessagingSystem.MessageHandlers.Length; index++)
```

Comment:
A batch message seems more appropriate - why are you going per-handler?

Reply:
I aimed for a minimal change. I really want people to be able to not care about those messages. So, they must not impact anything, almost always.
I felt the OrderingMessage would be more complex and harder to debug if it had variable size. With all the edge conditions: too many message types -> then the OrderingMessage overflows, etc. This is a one-off at connection type so I went with simple/stupid design. 

Willing to reconsider if you insist.